src/com/content-games/grid-selector.js

```diff
@@ -0,0 +1,81 @@
+import { h, Component } 				from 'preact/preact';
+
+import Dropdown							from 'com/input-dropdown/dropdown';
+import ButtonBase						from '../button-base/base';
+import SVGIcon 							from 'com/svg-icon/icon';
+
+export default class GridSelector extends Component {
+    constructor( props ) {
+		super(props);
+        this.state = {
+            expanded: false,
+            selected: null,
+        };
```

Comment:
Please wrap member names in string quotes. i.e. `{'expanded': false, ...}`. For consistency with JSON.